src/amo/reducers/collections.js

```diff
@@ -315,14 +316,16 @@ type LoadCurrentCollectionPageAction = {|
 
 export const loadCurrentCollectionPage = ({
   addons,
+  numberOfAddons,
 }: LoadCurrentCollectionPageParams = {}): LoadCurrentCollectionPageAction => {
-  if (!addons) {
-    throw new Error('addons are required');
+  invariant(addons, 'The addons parameter is required');
+  if (typeof numberOfAddons !== 'number') {
```

Comment:
We can still use `invariant` here, it would look like:

`invariant(typeof numberOfAddons === 'number', 'The numberOfAddons parameter must be a number');`

Reply:
Oh! Yes. It looks better, thanks.